pubspec.yaml

```diff
@@ -56,6 +56,8 @@ dependencies:
   logger: ^1.4.0
   sembast: ^3.4.9
   sembast_web: ^2.1.3
+  build_runner: ^2.4.1
```

Comment:
build_runner should not be a dependency here - it is a dev dependency used for code generation, but shouldn't be included in the compiled app